### DISCUSSION TITLE
500 error when loading /livewire/livewire.js

### DISCUSSION THREAD

**Discussion by derekcavaliero** [livewire/livewire#5997] [1. Help]
This is an odd one.

I am using `Laravel v10.10` and `Livewire v3.0@beta`. 

Everything works fine on localhost, files load without issue. But once I deploy to our RunCloud/DigitalOcean environment I start getting 500 errors when the browser tries to download the `/livewire/livewire.js?id=xxxxxxxx` asset.

Looking at my Sentry logs it seems to be throwing the following error:
 
![Screenshot 2023-07-26 at 7 08 59 AM](https://github.com/livewire/livewire/assets/5420084/d0c471d5-8203-429c-aff0-29110180775a)

Any idea what might be going on? I don't get errors for anything else - the only thing added was the Livewire package.

**Comment by nuernbergerA** [comment]
@derekcavaliero pls run `composer show symfony/http-foundation` and post the output here pls

**Comment by derekcavaliero** [comment]
Sure thing, here's a screenshot of the output @nuernbergerA 

![SCR-20230726-jkpu](https://github.com/livewire/livewire/assets/5420084/7074fa9f-096f-4d90-bea2-e37ed8c09cf3)

**Reply by nuernbergerA** [reply]
do you cache your routes in the environment where you deploy to?

**Reply by derekcavaliero** [reply]
Yes we had been, I noticed another issue that might be related to this: https://github.com/livewire/livewire/discussions/6000

That being said, even if I cleared all of the application caches in the production server (and didn't rebuild them), we still run into the same issue.

**Reply by nuernbergerA** [reply]
yeah there are multiple issues open about caching and the team is working on it

**Reply by derekcavaliero** [reply]
Yep, I see that. I do wonder if this is a separate issue though. 

I don't have caching enabled locally and the Livewire assets load without issue - but this is not true in production when I clear the cache there (and do not rebuild it).
